test/unit/lib/plugins/plugin/install.test.js

```diff
@@ -62,7 +66,7 @@ describe('PluginInstall', () => {
     let installStub;
 
     beforeEach(() => {
-      installStub = sinon.stub(pluginInstall, 'install').returns(BbPromise.resolve());
+      installStub = sinon.stub(pluginInstall, 'install').returns(Promise.resolve());
```

Comment:
can we replace it with `.resolves` instead of `returns(Promise.resolve())`? It is repeated in a few other places as well